easybuild/framework/easyconfig/tools.py

```diff
@@ -546,3 +545,6 @@ def dump_env_script(easyconfigs):
         print_msg("Script to set up build environment for %s dumped to %s" % (ecfile, script_path), prefix=False)
 
         restore_env(orig_env)
+
+
+from easybuild.tools.github import fetch_easyconfigs_from_pr, download_repo
```

Comment:
@Caylo we need to find a way around this (I'm even surprised it help in working around a cyclic import at all)